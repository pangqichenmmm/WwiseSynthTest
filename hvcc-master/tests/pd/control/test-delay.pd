#N canvas 723 255 388 397 10;
#X obj 38 41 loadbang;
#X obj 153 253 t b f;
#X obj 226 165 t b b;
#X msg 253 247 stop;
#X obj 226 225 t b b;
#X obj 38 93 t b b b;
#X obj 228 55 delay 1000 1 min;
#X obj 229 31 delay 1000 1 sec;
#X obj 228 78 delay 1000 1 samp;
#X obj 153 345 print;
#X msg 38 252 10;
#X obj 38 277 delay 5 1 msec;
#X msg 153 227 8;
#X obj 153 277 del 5;
#X obj 226 277 delay 10;
#X obj 226 195 delay 5;
#X obj 303 283 delay;
#X connect 0 0 5 0;
#X connect 1 0 13 0;
#X connect 1 1 13 1;
#X connect 2 0 15 0;
#X connect 2 1 14 0;
#X connect 3 0 14 0;
#X connect 4 0 14 0;
#X connect 4 1 3 0;
#X connect 5 0 10 0;
#X connect 5 1 12 0;
#X connect 5 2 2 0;
#X connect 10 0 11 0;
#X connect 11 0 9 0;
#X connect 12 0 1 0;
#X connect 13 0 9 0;
#X connect 14 0 9 0;
#X connect 15 0 4 0;
