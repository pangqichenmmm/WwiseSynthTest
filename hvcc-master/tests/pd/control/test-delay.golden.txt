[@ 8.000] print: bang
[@ 10.000] print: bang
[@ 15.000] print: bang
